pages/18f/how-18f-works/research-guidelines.md

```diff
@@ -171,32 +178,35 @@ In 2022, TTS considers $70/hour as a starting point for compensation. Teams
 should consider factors of equity and complexity of the research when
 determining compensation for user research on their project.
 
-### How do I actually distribute the compensation to research participants?
+## How do I actually distribute the compensation to research participants?
 
 There are a few different ways to distribute compensation to research
 participants.
 
-**Pay participants via gift card:** Compensation can be distributed to
-participants via digital/e-gift or physical gift cards through
-[GiftBit](https://www.giftbit.com/). Gift cards can be purchased by submitting a
+### Pay participants via gift card
+
+Compensation can be distributed to participants via digital/e-gift or physical
+gift cards through [GiftBit](https://www.giftbit.com/). Gift cards can be
+purchased by submitting a
 [purchase card request](https://docs.google.com/forms/d/e/1FAIpQLSd-GoOE9xWWfJvdZNRP3SE7mj5ysI_RfM8brxdG8YpyJV9yKA/viewform).
 The total amount of the purchase must be under $10,000 per project.
 
 For more information, see
 [how do I complete a micropurchase request?]({{site.baseurl}}/office-of-acquisition/#how-do-i-complete-a-micropurchase)
 in the handbook.
 
-**Use usertesting.com:** Teams can also use usertesting.com, if that’s
-applicable. TTS currently has a no cost agreement for that service (with help
-from a third party vendor)
+### Use usertesting.com
+
+Teams can also use usertesting.com, if that’s applicable. TTS currently has a no
+cost agreement for that service (with help from a third party vendor)
 [per instructions](https://docs.google.com/document/d/1zJuzKW1txg73ukZlFF37vp5RRhf-iR0r2QZzs0-MXY8/edit).
```

Comment:
One issue that might be helpful is labeling these links and docs more clearly.  Its unclear if there multiple "per instructions" or if there are multiple links to the same information. I feel overwhelmed with the number of links and confused on what to click on first. ... This also brings me to the issue of the person reading this having to hop back and forth between this handbook pages and multiple docs. Maybe there needs to be one doc will all the locked information in it so that a person is opening one and not multiple docs!